api/metrics-api-util/src/main/java/org/hawkular/metrics/api/jaxrs/config/ConfigurationKey.java

```diff
@@ -40,7 +40,9 @@
             "CASSANDRA_MAX_REQUEST_CONN", false),
     WAIT_FOR_SERVICE("hawkular.metrics.waitForService", null, null, true),
     DEFAULT_TTL("hawkular.metrics.default-ttl", "7", "DEFAULT_TTL", false),
-    DISABLE_METRICS_JMX("hawkular.metrics.disable-metrics-jmx-reporting", null, "DISABLE_METRICS_JMX", true);
+    DISABLE_METRICS_JMX("hawkular.metrics.disable-metrics-jmx-reporting", null, "DISABLE_METRICS_JMX", true),
+    METRICS_PUBLISH_DELAY("hawkular-metrics.publish-delay", "1000", "METRICS_PUBLISH_DELAY", false),
+    METRICS_PUBLISH_PERIOD("hawkular-metrics.publish-period", "2000", "METRICS_PUBLISH_PERIOD", false);
```

Comment:
I missed your review on the PR @jshaughn, sorry, responding now.
This 2s delay is in line with the period used in alerting for computing, so it is configurable to tweak it as result of performance tests.
I don't see it needs to be increased in our business domain, we are not doing real time alerting.
For example, MiQ has a default evaluation time of 10 minutes.
Having said that, the implementation lets you to set the value you need.